src/main/java/it/pagopa/pagopa/apiconfig/entity/Pa.java

```diff
@@ -32,7 +29,7 @@
     @Column(name = "RAGIONE_SOCIALE")
     private String ragioneSociale;
 
-    @Column(name = "FK_INT_QUADRATURE")
+    @JoinColumn(name = "FK_INT_QUADRATURE")
```

Comment:
without a mapping relationship ? is it useful ?

Reply:
I thought it will be useful to us in the future. Is it better to revert this change?

Reply:
no let's leave it. it was just to understand better